vic/drivers/shared_image/src/check_domain_info.c

```diff
@@ -0,0 +1,81 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Routines to compare the global domain to the other VIC input files
```

Comment:
This line seems a little vague- maybe add a little more description (e.g. what input files?; why are you doing the comparison?)